setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import find_packages, setup
+
+from version import get_version
+
+setup(
+    name="digital-land-data-manager",
+    version=get_version(),
+    author="MHCLG Digital Land Team",
```

Comment:
This should probably read DLUHC.

Reply:
:+1: I copied a lot of this file verbatim from `digital-land-python`